pulsar-websocket/src/main/java/com/yahoo/pulsar/websocket/AbstractWebSocketHandler.java

```diff
@@ -81,10 +81,14 @@ public void onWebSocketConnect(Session session) {
                     log.warn("[{}] WebSocket Client [{}] is not authorized on topic {}", session.getRemoteAddress(), role,
                             topic);
                     close(WebSocketError.NotAuthorizedError);
+                } else {
+                    createClient(session);
```

Comment:
As mentioned [here](http://download.eclipse.org/jetty/stable-9/apidocs/org/eclipse/jetty/websocket/api/WebSocketAdapter.html), `onWebSocketConnect` seems to be called after handshake is completed and `super.onWebSocketConnect(session)` seems to only set member variables.
In addition, as you stated in #98, `onWebSocketText(..)` gets triggered at server only when server releases thread from `onWebSocketConnect()`.
I think we need not move `super.onWebSocketConnect(session)`.
What do you think?